rust/src/api/mod.rs

```diff
@@ -1 +1,2 @@
 pub mod get_node_by_path;
+pub use get_node_by_path::get_node_by_path;
```

Comment:
```suggestion
```

how about we do this on the `lib.rs` instead, so the code in `lib.rs` will be:

```rs
mod _auto_generated;
mod api;
mod node;

pub use api::get_node_by_path;
pub use node::model::{Node, NodeName, NodeTerms, NodeType};

```

Reply:
you mean?
```rs
pub use api::get_node_by_path::get_node_by_path;
```